dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/SfdrDataController.kt

```diff
@@ -0,0 +1,41 @@
+package org.dataland.datalandbackend.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.swagger.v3.oas.annotations.Operation
+import org.dataland.datalandbackend.interfaces.DataManagerInterface
+import org.dataland.datalandbackend.interfaces.DataMetaInformationManagerInterface
+import org.dataland.datalandbackend.model.CompanyAssociatedData
+import org.dataland.datalandbackend.model.DataMetaInformation
+import org.dataland.datalandbackend.model.sfdr.SfdrData
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * Controller for the Sfdr framework endpoints
+ * @param myDataManager data manager to be used
+ * @param myObjectMapper object mapper used for converting data classes to strings and vice versa
+ */
+@RequestMapping("/data/sfdr")
+@RestController
+class SfdrDataController(
+    @Autowired var myDataManager: DataManagerInterface,
+    @Autowired var myMetaDataManager: DataMetaInformationManagerInterface,
+    @Autowired var myObjectMapper: ObjectMapper
+) : DataController<SfdrData>(
+    myDataManager,
+    myMetaDataManager,
+    myObjectMapper,
+    SfdrData::class.java
+) {
+    @Operation(operationId = "getCompanyAssociatedSfdrData")
```

Comment:
The operationId is only setting the name of the function in the auto generated client code to prevent having always the same names for functions with 1, 2, 3 ... attached to it. Everything else is not affected.